earth_enterprise/src/fusion/portableglobe/servers/fileunpacker/Linux/build_lib

```diff
@@ -1,5 +1,5 @@
 cp ../shared/* .
 swig -python -c++ glc_unpacker.i
-g++ -O2 -fPIC -I/usr/include/python2.7 -c glc_unpacker_wrap.cxx
-g++ -O2 -c -fPIC glc_unpacker.cpp portable_glc_reader.cpp file_unpacker.cpp file_package.cpp packetbundle.cpp packetbundle_finder.cpp
-g++ -shared glc_unpacker_wrap.o glc_unpacker.o portable_glc_reader.o file_unpacker.o file_package.o packetbundle.o packetbundle_finder.o -o _glc_unpacker.so
+g++ -std=gnu++98 -O2 -fPIC -I{python_inc_dir} -c glc_unpacker_wrap.cxx
```

Comment:
Some of these changes have also been made on the master branch.  Would it be simpler to merge master into centos-6-support?

Reply:
This uses the commits from the 5.2.0 branch, which were merged into `master`. When this branch is being merged into `master` Git should figure out that the changes on the two branches being merged are from the same commit and automerge.